<?xml version="1.0"?>
<launch>
    <node pkg="mcr_manipulation_measurers" type="component_wise_pose_error_calculator_node"
          name="component_wise_pose_error_calculator" output="screen">

        <!-- the cycle rate of the node (in seconds) -->
        <param name="loop_rate" type="double" value="0.1"/>

    </node>
</launch>